ccgains/trades.py

```diff
@@ -560,17 +560,19 @@ def append_binance_csv(
             self, file_name, which_data='trades', delimiter=',',
             skiprows=1,  default_timezone=tz.tzutc()):
```

Comment:
Everything good so far, but could you add a docstring here please? Just copy & adapt the one from append_poloniex_csv, then you'd merely have to add 'distribution' to the explanation of `which_data`.